jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -286,7 +286,8 @@ public void generate(Path targetDirectory) throws IOException {
         // '/app/classes/com/test/HelloWorld.class', the resolved destination would be
         // 'target/jib-docker-context/classes/com/test/HelloWorld.class'.
         Path baseExtractionPath = Paths.get(copyDirective.extractionPath.toString());
-        Path relativeEntryPath = baseExtractionPath.relativize(layerEntry.getExtractionPath());
+        Path relativeEntryPath =
+            baseExtractionPath.relativize(Paths.get(layerEntry.getExtractionPath().toString()));
```

Comment:
I think we can now use `copyDirective.extractionPath.resolve(relativeEtryPath)`.

Reply:
I'm not sure what you mean here - the `copyDirective.extractionPath` is converted into the `baseExtractionPath` in order to relativize against the `layerEntry.getExtractionPath`. I decided not to implement the relativize method on `AbsoluteUnixPath` here since it seemed overkill for this one scenario.

Reply:
Ah, thought it was `resolve` than `relativize`.